README.md

```diff
@@ -2,51 +2,126 @@
-  with the space, e.g. `long_run=t1.test.lua t2.test.lua`)
-* `config` - test configuration file name
-
-Field `core` must be one of:
-
-* `tarantool` - Test-Suite for Functional Testing
-* `app` - Another functional Test-Suite
-* `unittest` - Unit-Testing Test Suite
-
-### Test
-
-Each test consists of files `*.test(.lua|.sql|.py)?`, `*.result`, and may have
-skip condition file `*.skipcond`.  On first run (without `.result`) `.result`
-is generated from output.  Each run, in the beggining, `.skipcond` file is
-executed. In the local env there's object `self`, that's `Test` object. If test
-must be skipped - you must put `self.skip = 1` in this file. Next,
-`.test(.lua|.py)?` is executed and file `.reject` is created, then `.reject` is
-compared with `.result`. If something differs, then 15 last string of this diff
-file are printed and `.reject` file is saving in the `<vardir>/rejects/<suite>`
-subfolder given in options or set localy as `var/rejects/<suite>` by default.
-If not, then `.reject` file is deleted.
+## Test suite configuration
+
+Test suite is a bunch of tests located in a subdirectory (recursively) with a `suite.ini`
+file. The `suite.ini` is a basic ini-file with one section `[default]`.
+
+```ini
+[default]
+core = luatest
+description = Example test suite using luatest
+...
+```
+
+There's a syntax for single- and multiline lists. (TODO see ./sql-tap/suite.ini)
+For example:
+
+```ini
+single_line = first.test.lua second.test.lua third.test.lua
+multiline = first.test.lua ;
+            second.test.lua ;
+            third.test.lua ;
+            ...
+```
+
+Below is a list of configuration values (fields) in the `suite.ini` file.
+
+### General configuration
+
+* `core` — major testing dependency or method
+  Should have one of the following values:
+
+  * `tarantool` — test suite used for functional testing
+  * `app` — TAP-tests, another functional test suite
+  * `unittest` — unit testing test suite
+  * `luatest` — test suite using luatest library
+
+* `description` — test suite description
+* `script` (optional) — shebang file to start tarantool with.
+  Most often, the value is a file in the same directory as `suite.ini`, like `box.lua` or `master.lua`.
+  
+* `config` — name of a test configuration file, for example, `engine.cfg`.
+  TODO: what's a test configuration file?
+
+* `lua_libs` — paths to Lua files that should be copied to the directory,
+  where a server is started. For example:
+
+  ```ini
+  lua_libs = lua/require_mod.lua lua/serializer_test.lua lua/process_timeout.lua
+  ```
+
+### Skipping tests
+
+A number of fields are used to disable (skip) certain tests:
+
+* `disabled` — list of tests that should be skipped
+* `release_disabled` — list of tests that should be skipped when Tarantool is
+    built with `Release`
+* `valgrind_disabled` — list of tests that should be skipped when Valgrind is enabled
+* `long_run` — mark tests as long, such tests will run only with `--long` option.
```

Comment:
It would be good to give a suggestions, what is considered 'short' and what 'long'. A long time ago the following informal rule was spoken: a test should usually pass within 1 second. Now it is far away of truth. Maybe not a question for this PR.